packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -0,0 +1,549 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import withStyles from '../../../material-ui/src/styles/withStyles';
+import ButtonBase from '../../../material-ui/src/ButtonBase';
+import { fade } from '../../../material-ui/src/styles/colorManipulator';
+import clamp from '../utils/clamp';
```

Comment:
I meant for this to be in the core `material-ui` so we can reuse it there, but I can fix it up later.